src/pages/contact.md

```diff
@@ -7,10 +7,14 @@ description: Contact Camunda
 
 There are various channels where you can reach us.
 
-- We encourage everyone to participate in our **community chat** via [Slack](https://zeebe-io.slack.com/), where you can exchange ideas with other Zeebe and Camunda Cloud users, as well as the product developers.
+- We encourage everyone to participate in our **community** via the [Camunda Cloud community forum](https://forum.camunda.io/) or [Slack](https://camunda-cloud.slack.com/), where you can exchange ideas with other Zeebe and Camunda Cloud users, as well as the product developers.
 
-- If you find a **bug**, please open an issue in this [repository](https://github.com/camunda-cloud/bugs). The bugs have a default template, so please describe what doesn't work in detail.
+- If you find a **bug**, please open an issue in this [repository](https://github.com/camunda-cloud/issues). The bugs have a default template, so please describe what doesn't work in detail.
 
-- For **security-related issues**, please e-mail us at [this address](mailto:cloud-security@camunda.com) so we can solve the problem as quickly as possible.
+- We also accept **feature requests** through a similar process. Please open an issue in this [repository](https://github.com/camunda-cloud/issues). The feature requests have a default template, so please describe what you'd like to see in detail.
 
-- **Feebdack and Support** can be submitted or requested via the corresponding entry in the [Cloud Console navigation menu](../../docs/product-manuals/cloud-console/troubleshooting/feedback-and-support). If you have a **license agreement** with us you will be redirected to the [Support queue](https://jira.camunda.com/projects/SUPPORT/) at Camunda. In the Free Plan, contact can be made using the internal form:
+- For **security-related issues**, please e-mail us at [this address](mailto:cloud-security@camunda.com) so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
+
+- **Feedback and Support** can be submitted or requested via the corresponding entry in the [Cloud Console navigation menu](../../docs/product-manuals/cloud-console/troubleshooting/feedback-and-support). 
```

Comment:
I guess we can save the user some time, the feedback and support button leads to forum for trail users, and JIRA for the paid users. Why aren't we linking directly to the cloud forum here, and then add half-sentence like "if you are an enterprise customer, please see the enterprise documentation" with a link to the enterprise docs. I guess we still have to create the enterprise docs?

Also a second thought, pointing people to camunda cloud console is strange for self managed and cloud enterprise on-prem users, which might not have an account in SaaS.

Reply:
> I guess we still have to create the enterprise docs?

That's the exact question I was asking myself. 

For launch, should we clearly mentioned 2 paths for support, forum and JIRA? The JIRA path is creating a ticket in the Support project, right?

Reply:
Yes the support (paid customer) path is to create an Jira issues in the Support project.

Reply:
- **Feedback and Support** can be submitted or requested via JIRA for subscription or enterprise customers in the Support project. Otherwise, please use the community forum. For more information about Enterprise support and additional support resources please see [Enterprise Support](https://camunda.com/support/).

^ Would something like that work for now?

Reply:
yes sounds good to me